llvm/test/CodeGen/AMDGPU/datalayout-non-zero-lds-value.ll

```diff
@@ -0,0 +1,20 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt < %s -S -mtriple=amdgcn-- | FileCheck %s
```

Comment:
I don't know what this test is supposed to be showing

Reply:
This is a positive test case to check if the data layout string related sentinel pointer value is parsed correctly.